pkg/apis/projectcalico/v3/register.go

```diff
@@ -30,7 +31,7 @@ func init() {
 
 // Adds the list of known types to api.Scheme.
 func addKnownTypes(scheme *runtime.Scheme) error {
-	all := []runtime.Object{
+	AllKnownTypes = []runtime.Object{
```

Comment:
Can we define this outside the function since this is reliant on us never removing addKnownTypes from the init().